example/cln_grpc_example.dart

```diff
@@ -1,5 +1,6 @@
 import 'package:cln_common/cln_common.dart';
 import 'package:cln_grpc/cln_grpc.dart';
+import 'package:fixnum/fixnum.dart';
```

Comment:
we really need this?

Reply:
Yes, We need to declare msatoshi with the type `Amount`, so to convert integer to `Int64` we need it

Here ```Int64 msat = Int64(1000);```

Reply:
Cool! more deps hahah